pkg/utils/miniojob/types.go

```diff
@@ -172,13 +172,17 @@ func (jobCommand *MinIOIntervalJobCommand) createJob(_ context.Context, _ client
 		},
 	}
 	baseEnvFrom = append(baseEnvFrom, jobCommand.CommandSpec.EnvFrom...)
+	urlProtocol := "http://"
+	if isTLS {
+		urlProtocol = "https://"
+	}
 	secret := &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      fmt.Sprintf("%s-job-secret", jobCR.Name),
 			Namespace: jobCR.Namespace,
 		},
 		StringData: map[string]string{
-			"MC_HOST_myminio":                    fmt.Sprintf("https://$(ACCESS_KEY):$(SECRET_KEY)@minio.%s.svc.cluster.local", jobCR.Namespace),
+			"MC_HOST_myminio":                    fmt.Sprintf("%s$(ACCESS_KEY):$(SECRET_KEY)@minio.%s.svc.cluster.local", urlProtocol, jobCR.Namespace),
```

Comment:
```suggestion
			"MC_HOST_myminio":                    fmt.Sprintf("%s://$(ACCESS_KEY):$(SECRET_KEY)@minio.%s.svc.%s", scheme, jobCR.Namespace, v2.GetClusterDomain()),
```
Please use `v2.GetClusterDomain()` instead of `cluster.local`. I would have even prefered to use `v2.MinIOFQDNServiceName()`, but that's a bit hard to get the access-key and secret-key in there.

Reply:
Also change it on the next line for the `MC_STS_ENDPOINT_myminio`.